oca_port/cli/main.py

```diff
@@ -15,10 +15,17 @@
     $ export GITHUB_TOKEN=<token>
     $ oca-port 13.0 14.0 shopfloor --verbose
 
-To effectively migrate the addon or port its commits, use the `--fork` option:
+To effectively migrate the addon or port its commits, use the `--destination` option:
 
-    $ oca-port 13.0 14.0 shopfloor --fork camptocamp
+    $ oca-port 13.0 14.0 shopfloor --destination camptocamp/wms#14-mig-shopfloor
```

Comment:
Is there a commonly accepted standard to write this? `git` itself sometimes use `{remote}:{branch}` syntax, there is also `@` used to mention a reference, like in `git+https://github.com/OCA/oca-port.git@refs/pull/26/head`.

Should we support several patterns?

- `{org}/{repo}#{branch}`
- `{org}/{repo}@{branch}`
- `{org}/{repo}:{branch}`